### ISSUE TITLE
Wall Thermostat API incomplete

### ISSUE THREAD

**Issue by zabeloliver** [BoschSmartHome/bosch-shc-api-docs#77]
Hi,
while playing with the API and my WallThermostats (230V Floorheating) it seems, that the current heating state is not included in the Api. It is not possible to see whether the thermostat / heating is on or off. Intresstingly if you look in Apple HomeKit, you can actually see the state.

Do I miss something or is the state not available?
Thanks for clarifying!

Best regards, 
Oli

**Comment by simont77** [comment]
I confirm the same also with new RoomThermostat II 230V. State appears on Homekit, but in the API response there is not any State that we can referred to the actual heating state. Only actual temperature, setpoint and actual humidity.

**Comment by sebastianharder** [comment]
Are you referring to the "Heating break & cooling" setting in the Bosch Smart Home app?

**Comment by zabeloliver** [comment]
No, we are referring to the state, whether the heating is currently active (valve is open) or not active (valve is closed). HomeKit shows this state, but I can’t see it in the APIs response. Better would be of course an percentage value, but on / off would also be ok ;)

**Comment by NoPlayBack** [comment]
Is there any chance that the current state of the RoomThermostat II 230V will be added as value somewhere in the API ?
Or is it available already, as there are rumours that Apple HomeKit is able to display that value?

That information would be very valuable in order to understand and optimize the heating behaviour of one room... and to calculate the actual requested amount of heating energy in the building...

**Comment by patrickdawson** [comment]
I am interested in that feature too. I just created an issue in [node-red-contrib-bosch-shc](https://github.com/hxmelab/node-red-contrib-bosch-shc) when I realized there is already an open issue here and that info is not even available in the Bosch REST API.
I can confirm that this info is available in Apple HomeKit: When the heating is active the temperature number turns orange. It is black when the heating is switched off.

**Comment by georg-un** [comment]
I would be very interested in this feature too. I don't understand why this information isn't displayed in the app in the first place. As a user, I don't even know what the thermostat is doing right now. This makes it very hard for me to find a heating/cooling schedule that actually saves me money which defeats the whole purpose of the thermostat being smart.

**Comment by mosandlt** [comment]
Same root cause as #121: the Room/Wall Thermostat (THB) doesn't drive a valve directly, so it has no `ValveTappetState`/actuator-position field of its own — that data lives on the actual TRV radiator thermostat(s) `RoomClimateControl` is controlling for that room. We can confirm on our system that `ValveTappetState` is present and correct via `ThermostatService` on every one of our 13 TRV/TRV_GEN2 devices. If HomeKit is showing "heating active" for a room with only a wall thermostat, it's likely aggregating the child TRVs' valve state rather than reading anything from the wall thermostat's own services — worth checking whether the room in question actually has TRVs behind it.
